cvmfs/ssl.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#ifndef CVMFS_SSL_H_
+#define CVMFS_SSL_H_
```

Comment:
A top level comment should explain that this is about CA certificates of servers.  There is some other code in authz that deals with client certificates.